index.d.ts

```diff
@@ -24,22 +25,40 @@ declare module 'plaid' {
     offset?: number;
   }
 
+  interface AssetReportUser {
+    client_user_id?: string;
+    first_name?: string;
+    middle_name?: string;
+    last_name?: string;
+    ssn?: string;
+    phone_number?: string;
+    email?: string;
+  }
+
+  interface AssetReportCreateOptions {
```

Comment:
if these are options, shouldn't the fields be optional @dsfish ?

Reply:
⚡️